adm/_metwork.spec

```diff
@@ -393,15 +410,22 @@ EOF
             if test -d /home/{{MFMODULE_LOWERCASE}}; then
                 echo "INFO: saving old /home/{{MFMODULE_LOWERCASE}} to /home/{{MFMODULE_LOWERCASE}}.${SAVE_SUFFIX} ..."
                 mv /home/{{MFMODULE_LOWERCASE}} /home/{{MFMODULE_LOWERCASE}}.${SAVE_SUFFIX}
+                mkdir /home/{{MFMODULE_LOWERCASE}}
+                #Keep .ssh directory for the next install
+                if test -d /home/{{MFMODULE_LOWERCASE}}.${SAVE_SUFFIX}/.ssh; then
+                    cp -Rp /home/{{MFMODULE_LOWERCASE}}.${SAVE_SUFFIX}/.ssh /home/{{MFMODULE_LOWERCASE}}
+                fi
                 rm -Rf /home/{{MFMODULE_LOWERCASE}}.${SAVE_SUFFIX}/log
                 rm -Rf /home/{{MFMODULE_LOWERCASE}}.${SAVE_SUFFIX}/tmp
                 {% if MFMODULE == "MFDATA" %}
                     rm -Rf /home/{{MFMODULE_LOWERCASE}}.${SAVE_SUFFIX}/var/in
                 {% endif %}
-                mkdir /home/{{MFMODULE_LOWERCASE}}
+                chown -R --reference=/home/{{MFMODULE_LOWERCASE}}.${SAVE_SUFFIX} /home/{{MFMODULE_LOWERCASE}}
+                chmod --reference=/home/{{MFMODULE_LOWERCASE}}.${SAVE_SUFFIX} /home/{{MFMODULE_LOWERCASE}}
             fi
-            userdel -f -r {{MFMODULE_LOWERCASE}} 2>/dev/null
-            rm -Rf /home/{{MFMODULE_LOWERCASE}} 2>/dev/null
+            #Keep /home/{{MFMODULE_LOWERCASE}} (almost empty) to keep permanent stuff such as .ssh directory
```

Comment:
remove useless comments !?

Reply:
Useful for me (it's a big change not to remove the directory and the user)

Reply:
OK